boofuzz/sessions.py

```diff
@@ -1162,6 +1169,8 @@ def transmit_fuzz(self, sock, node, edge, callback_data):
             else:
                 self._fuzz_data_logger.log_info(msg)
             pass
+        except (exception.BoofuzzTargetConnectionFailedError, exception.BoofuzzSSLError) as e:
+            self._fuzz_data_logger.log_fail(str(e))
```

Comment:
Same as comment in `transmit_normal()`.

Catching `BoofuzzTargetConnectionFailedError` not needed; maybe add an option to log SSL errors on info level.

Reply:
Please see my comment for transmit_normal() above.

Reply:
Fixed in commit b2b3cec